recipes/pylint-django/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pylint-django" %}
+{% set version = "0.7.2" %}
+{% set sha256 = "b8e51d6b694401e23040ddafc3a4c7da92faaa1718b007ad66aa36e40ec84506" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pylint-plugin-utils >=0.2.1
+    - pylint >=1.0
+
+test:
+  imports:
+    - pylint_django
+    - pylint_django.augmentations
+    - pylint_django.checkers
+    - pylint_django.transforms
+
+about:
+  home: https://github.com/landscapeio/pylint-django
+  license: GPL 2
```

Comment:
License file not yet in an `sdist`. Proposing it be added in PR ( https://github.com/landscapeio/pylint-django/pull/82 ).